mcs/class/System.IdentityModel/Makefile

```diff
@@ -7,11 +7,16 @@ OTHER_LIB_MCS_FLAGS = -r:System.Web.ApplicationServices.dll
 endif
 
 LIBRARY = System.IdentityModel.dll
-LIB_REFS = System System.Xml System.Security System.Web System.Configuration Mono.Security System.Runtime.Serialization
+LIB_REFS = System System.Xml System.Security System.Configuration Mono.Security System.Runtime.Serialization
 LIB_MCS_FLAGS = \
 		/d:NET_3_0	\
 	        $(OTHER_LIB_MCS_FLAGS)
 
+XAMMAC_4_5 := $(filter net_4_5_xammac, $(PROFILE))
+ifndef XAMMAC_4_5
```

Comment:
Maybe define NO_SYSTEM_WEB_DEPENDENCY in mcs/build/profiles/net_4_5_xammac.make and use a feature-define here as well.